src/checkers/inference/InferenceLauncher.java

```diff
@@ -37,6 +37,9 @@
     private final PrintStream outStream;
     private final PrintStream errStream;
 
+    private static final String PROP_PREFIX = "InferenceLauncher";
+    private static final String RUNTIME_BCP = PROP_PREFIX + ".runtime.bcp";
```

Comment:
Similar code in https://github.com/eisop/checker-framework/blob/master/framework/src/main/java/org/checkerframework/framework/util/CheckerDevelMain.java would call this `RUNTIME_BCP_PROP` to make clear it's a property key.